src/data/value/mod.rs

```diff
@@ -34,6 +35,8 @@ pub enum Value {
     Map(HashMap<String, Value>),
     List(Vec<Value>),
     Null,
+    #[serde(with = "serde_str")]
```

Comment:
It will work without this above.

Not important though, could you locate `Decimal(Decimal)` to the below of `F64`?
```rust
..
enum Value {
    Bool(bool),
    I64(i64),
    F64(f64),
    Decimal(Decimal),
    Str(String),
    ..
}
..
```

`I64`, `F64` and `Decimal` are all numeric types, so it would be good to gather them.